chapters/usage-profile.md

```diff
@@ -0,0 +1,244 @@
+# 31 Usage Profile section
+
+## 31.1 Usage Scope field <a name="31.1"></a>
+
+### 31.1.1 Description
+
+Software packages are sometimes generated with specific Build conditions based on package supplier's license conditions and intended usage in addition to original software source code. In those cases, it prefer to package supplier clearly state into the SPDX document that the intended usage of the package as envisioned when setting the Build conditions by the package supplier.
+In other cases, some specific licenses out of dual or multiple licenses of original software source code may be applied to the package according to the Build conditions. Or, the inspection conditions for packages may also have to be clearly stated for each package.
```

Comment:
Minor grammar suggestion:
`In those cases, it prefer to package supplier clearly state into the SPDX document that the intended usage of the package as envisioned when setting the Build conditions by the package supplier.`

->

`In those cases, it is preferred for the package supplier to clearly state in the SPDX document the intended usage of the package as envisioned when setting the Build conditions.`